src/Core/SharedKernels/Questionnaire/WB.Core.SharedKernels.Questionnaire/Categories/CategoriesItem.cs

```diff
@@ -11,7 +11,7 @@ public class CategoriesItem
         public string Text { get; set; } = String.Empty;
         public string? AttachmentName { get; set; }
 
-        public override bool Equals(object obj)
+        public override bool Equals(object? obj)

```

Comment:
not sure it handles correctly nulls like this one:
var x =  (CategoriesItem)null;

Reply:
obj != null &&  obj is CategoriesItem item  == obj is CategoriesItem item